styles/opendns-block-page/catppuccin.user.css

```diff
@@ -0,0 +1,207 @@
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Aopendns-block-page
+@description Soothing pastel theme for OpenDNS Content Filtering Page
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document domain('block.opendns.com') {
+    @media (prefers-color-scheme: light) {
+      :root {
+        #catppuccin(@lightFlavor, @accentColor);
+      }
+    }
+    @media (prefers-color-scheme: dark) {
+      :root {
+        #catppuccin(@darkFlavor, @accentColor);
+      }
+    }
+  
+    #catppuccin(@lookup, @accent) {
+      @rosewater: @catppuccin[@@lookup][@rosewater];
+      @flamingo: @catppuccin[@@lookup][@flamingo];
+      @pink: @catppuccin[@@lookup][@pink];
+      @mauve: @catppuccin[@@lookup][@mauve];
+      @red: @catppuccin[@@lookup][@red];
+      @maroon: @catppuccin[@@lookup][@maroon];
+      @peach: @catppuccin[@@lookup][@peach];
+      @yellow: @catppuccin[@@lookup][@yellow];
+      @green: @catppuccin[@@lookup][@green];
+      @teal: @catppuccin[@@lookup][@teal];
+      @sky: @catppuccin[@@lookup][@sky];
+      @sapphire: @catppuccin[@@lookup][@sapphire];
+      @blue: @catppuccin[@@lookup][@blue];
+      @lavender: @catppuccin[@@lookup][@lavender];
+      @text: @catppuccin[@@lookup][@text];
+      @subtext1: @catppuccin[@@lookup][@subtext1];
+      @subtext0: @catppuccin[@@lookup][@subtext0];
+      @overlay2: @catppuccin[@@lookup][@overlay2];
+      @overlay1: @catppuccin[@@lookup][@overlay1];
+      @overlay0: @catppuccin[@@lookup][@overlay0];
+      @surface2: @catppuccin[@@lookup][@surface2];
+      @surface1: @catppuccin[@@lookup][@surface1];
+      @surface0: @catppuccin[@@lookup][@surface0];
+      @base: @catppuccin[@@lookup][@base];
+      @mantle: @catppuccin[@@lookup][@mantle];
+      @crust: @catppuccin[@@lookup][@crust];
+      @accent-color: @catppuccin[@@lookup][@@accent];
+  
+      color-scheme: if(@lookup = latte, light, dark);
+  
+      ::selection {
+        background-color: fade(@accent-color, 30%);
+      }
+  
+      input,
+      textarea {
+        &::placeholder {
+          color: @subtext0 !important;
+        }
+      }
+  
+      background-color: @base;
+
+      body {
+        color: @text;
+
+        a {
+          &, &:hover, &:visited, &:active {
+            color: @accent-color;
+          }
+        }
+      }
+
+      :focus {
+        outline-color: @accent-color;
```

Comment:
this *should* be `outline: 2px solid @accent-color` because outline settings aren't set by default and just setting the color doesn't overwrite the browser's default styling but i can't because of the linter

Reply:
linter didn't complain so we good